paimon-core/src/main/java/org/apache/paimon/catalog/Catalog.java

```diff
@@ -262,6 +263,12 @@ default void repairCatalog() {
         throw new UnsupportedOperationException();
     }
 
+    /** Repair databases or tables separated by commas. */
+    @Experimental
+    default void repairDatabasesOrTables(String databaseOrTables) throws TableNotExistException {
```

Comment:
Can you just extract a util method for `repairDatabasesOrTables`?
I don't want to modify API of catalog again.

Reply:
OK，would extract the logic to a util method instead of modify catalog. Thanks @JingsongLi